### PR TITLE
Merge feature/packaging branch to main

### PR DESCRIPTION
**Pull request by staskut**
packaging is implemented and tested.

### CODE REVIEW

**Review by st3n — APPROVED** [review]

**Comment by st3n** [comment]
I think we need to do next in terms of this PR:
1. create storage folder. Load and dump data there.
2. create src folder and move our modules there
3. all related to packing in separate dir

**Comment by staskut** [comment]
> create storage folder. Load and dump data there.

 - agree, will do

As for two other points:

> create src folder and move our modules there 
> all related to packing in separate dir

- I was following suggestions we have in the course. Same here - https://betterscientificsoftware.github.io/python-for-hpc/tutorials/python-pypi-packaging/#creating-a-python-package

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+
+setup(
```
Comment (gregorytereshko) [inline setup.py:3]: добавь, пожалуйста, этот пакет в зависимости https://pypi.org/project/jaro-winkler/

**Review by gregorytereshko — CHANGES_REQUESTED** [review]

**Comment by st3n** [comment]
> > create storage folder. Load and dump data there.
> 
> * agree, will do
> 
> As for two other points:
> 
> > create src folder and move our modules there
> > all related to packing in separate dir
> 
> * I was following structure suggestions we have in the course. Same here - https://betterscientificsoftware.github.io/python-for-hpc/tutorials/python-pypi-packaging/#creating-a-python-package
>   So module files are in the similarly named subfolder, not src. and setup.py one level above.

pyexample  from this example is the our "src" folder.

**Comment by st3n** [comment]
please provide a tittle adn description for PR

File: setup.py (hunk 1 above)
Comment (gregorytereshko) [reply setup.py:3]: И это тоже, пожалуйста
`prompt_toolkit`

**Review by st3n — DISMISSED** [review]

**Review by gregorytereshko — DISMISSED** [review]

**Review by st3n — DISMISSED** [review]

**Review by st3n — APPROVED** [review]

**Pull request merged**